pyuvsim/tests/test_run.py

```diff
@@ -39,14 +38,15 @@ def goto_tempdir(tmpdir):
     os.chdir(cwd)
 
 
+@pytest.mark.filterwarnings("ignore:antenna_diameters are not set")
 @pytest.mark.parametrize('paramfile', ['param_1time_1src_testcat.yaml',
                                        'param_1time_1src_testvot.yaml'])
 @pytest.mark.parallel(2)
 def test_run_paramfile_uvsim(goto_tempdir, paramfile):
     # Test vot and txt catalogs for parameter simulation
     # Compare to reference files.
     uv_ref = UVData()
-    uv_ref.read_uvfits(os.path.join(SIM_DATA_PATH, 'testfile_singlesource.uvfits'))
+    uv_ref.read(os.path.join(SIM_DATA_PATH, 'testfile_singlesource.uvfits'))
```

Comment:
how difficult owuld it be just to set the antenna diameters in this file?

Reply:
Not difficult, it just requires re-writing the reference files. Antenna diameters aren't required metadata, so it didn't seem worth rewriting the files for.

Reply:
Ah, I just looked at more context and I see why you're asking, because of the bug in the antenna numbers. I could rewrite it.

Reply:
I re-wrote the file and removed all the code fixing the object.